CHANGES.md

```diff
@@ -136,6 +136,12 @@ OpenSSL 3.0
 
    *Richard Levitte*
 
+ * Project text documents not yet having a proper file name extension
+   (HACKING, LICENSE, NOTES*, README*, VERSION) have been renamed to *.md
```

Comment:
> quote the glob patterns so that they are not seen as emphasis markers.

All right, done, also for several older entries,
although MD seems to cope with them well.